tests/compiler/builtins.ts

```diff
@@ -475,7 +475,7 @@ assert(lengthof<() => void>() == 0);
 assert(lengthof<(a: i32) => void>() == 1);
 assert(lengthof<(a: i32, b: C) => void>() == 2);
 assert(lengthof<(a: i32, b: C, c: string) => void>() == 3);
-assert(lengthof((a: i32, b: i32, c: i32, d: i32): void => {}) == 4);
+assert(lengthof((a: i32, b: i32, c: i32, d: i32) => {}) == 4);
```

Comment:
I guess for `lengthof` / `nameof` you should leave return type

Reply:
I can do that. But should I add a special case for those functions in `visitArrowFunction` so the linter doesn't complain about the unnecessary return type?

Reply:
yeah. will be great. @dcodeIO wdyt?

Reply:
Commit added.

Reply:
The return type if necessary in these cases if it cannot be inferred, i.e. the parameter's type is `any`, `unknown` or generic and generic parameters have been omitted.